package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "onboarding-servlerless-cdk-students",
+  "name": "Serverless_Flavio",
```

Comment:
Why did you change this name?

Reply:
I don't remember clearly, but I think it happened when I tried to clone the forked repository. I still have some difficulties working with GitHub. First, I choe the directory "onboarding-servlerless-cdk-students", and something went wrong. Then I chose another directory, which I named "Serverlees_Flavio".

Reply:
I understood it. But even if renaming the directory you don't need to change this attribute name. You can undo this change :)